File: DemoMethods/Advanced/ReplicationFailover.cs
using System;
using System.Linq;
using System.Web.Http;
using DemoMethods.Entities;
using DemoMethods.Helpers;
using DemoMethods.Indexes;
using Raven.Client;
using Raven.Client.Document;

namespace DemoMethods.Advanced
{
    public partial class AdvancedController : DemoApiController
    {
        [HttpGet]
        [Demo("ReplicationFailover", DemoOutputType.Flatten, demoOrder: 200)]
        public object ReplicationFailover(string id = "Users/1")
        {
            using (var store = new DocumentStore
            {
                Url = "http://localhost:8080",
                DefaultDatabase = "Rep1"
            }.Initialize())
            {
                using (var session = store.OpenSession())
                {
                    RavenQueryStatistics stats;
                    var results = session.Load<dynamic>(id);

                   
                    return results;
                }
            }
        }
    }
}

File: DemoServer/PythonDemo/demo_methods/Advanced/ReplicationFailover.py
from pyravendb.store.document_store import documentstore


class ReplicationFailover(object):
    @staticmethod
    def replication_failover(document_id):
        with documentstore(url="http://localhost:8080", database="Rep1") as store:
            store.initialize()

            with store.open_session() as session:
                results = session.load(document_id, object_type=dict)

                return results
